source/layouts/partials/_meta.html.erb

```diff
@@ -116,12 +116,11 @@
 <meta name="twitter:site:id" content="<%= config[:twitter_site_id] %>">
 
 <%# Links; next/prev should be canonical. %>
-<link rel="alternate" type="application/rss+xml" title="Sharesight RSS Feed" href="<%= base_url('/blog/feed.xml') %>">
```

Comment:
I think you still need to link to this, even though the link no longer exists here…it exists in Gatsby.